weed/topology/store_replicate_test.go

```
@@ -53,3 +65,101 @@ func TestDistributedOperationEmpty(t *testing.T) {
+	return &master_pb.LookupVolumeResponse{VolumeIdLocations: vls}, nil
+}
+
+// TestReplicatedWriteForwardsFsyncToReplicas verifies that the fsync=true
+// request parameter is forwarded to replica volume servers in the fan-out
+// request, so a durable write means every replica has flushed to disk.
+func TestReplicatedWriteForwardsFsyncToReplicas(t *testing.T) {
+	replicaQueries := make(chan url.Values, 4)
+	replica := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		replicaQueries <- r.URL.Query()
+		w.WriteHeader(http.StatusCreated)
+		w.Write([]byte(`{"size":1}`))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)